.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,6 +15,7 @@ PR.
 - [ ] Did you update relevant documentation?
 - [ ] Did you add tests to cover new code or a fixed issue?
 - [ ] Did you update the changelog?
+- [ ] Did you check off all checkboxes from the linked Linear task?
```

Comment:
Maybe a note about not needing to do this if you don't have linear access? Or do external devs have access to the task checkboxes somehow

Reply:
Addressed in 636090a